### PR TITLE
Move docker-composer.yml to dist version

### PR DESCRIPTION
**Pull request by jngermon**
To easily update repository without create conflict with personal docker-compose.yml file

### CODE REVIEW

**Comment by sameersbn** [comment]
What is the idea behind this PR?

Besides the `docker-compose.yml` file is referenced at multiple times in the README. This too would have to be updated.

**Comment by jngermon** [comment]
Your `docker-comose.yml` file is a sample file. It enable to test docker-gitlab but it contains default value.
I propose to use `docker-compose.yml.dist' file to show configuration sample. To use docmer-compose for gitlab I just have to copy `docker-compose.yml.dist` to `docker-compose.yml`. I can custom my file as I want and the modifications are ignored by git. My project is up-to-date and I can pull your commit without create conflict (because I don't update `docker-compose-.yml.dist` file).

**Comment by thomasf** [comment]
@jngermon  I think you should arrange your deployment configuration differently. You don't need anything else from this repository than the `docker-compose.yml`.

**Comment by jngermon** [comment]
The problem is still here. At the moment the `docker-comspose.yml` file contains parameters values (relative to deployment) and it's following by GIT. Basically, parameters files must to not be follow (except in dist version).
I'm working on my fork, I can create a special branch to change `docker-compose.yml` but I dont want to commit these changes (I don't want that my postgre password to be saved on Github and be public).
And if I don't commit changes on follow's file, I can't rebase on master.

**Comment by sameersbn** [comment]
While I agree with @jngermon, I also agree with @thomasf as I have the same issue as he describes but I just use `git stash` so that it does not bother me. 

For the benefit of new users, I decided that having the `docker-compose.yml` was a better choice so that they could simply do a `docker-compose up` after downloading the file. I am not opposed to renaming the file to `docker-compose.yml.dist` or `docker-compose.example.yml` as long as the instructions in the README are also updated to reflect that.

**Comment by jngermon** [comment]
Ok I understand. I'll add the instructions in the README before submit a new merge.

**Comment by jngermon** [comment]
I have changed the README file to add instructions to create your own docker-compose.yml file since the dist version.

File: README.md [hunk 1]
```diff
@@ -132,9 +132,19 @@ docker build --tag=$USER/gitlab .
...
+You just have to create your own `docker-compose.yml` file.
+Copy from our sample file
+```bash
+cp docker-compose.yml.dist docker-compose.yml
+```
+You can edit your `docker-compose.yml` file to change our default values.
+And now, you just have to run
+
+```bash
 docker-compose up
```
Comment (sameersbn) [inline README.md]: I think the instructions should be:

```bash
wget https://raw.githubusercontent.com/sameersbn/docker-gitlab/master/docker-compose.yml.dist -O docker-compose.yml
docker-compose up
```

Its a quickstart guide, copy-paste commands are preferred here :smile:

**Comment by jngermon** [comment]
Sorry for the delay.
I have done the modification that you suggested.

**Comment by sameersbn** [comment]
@jngermon this PR seems to have broken the deploy to tutum button. I am considering reverting this PR.

**Comment by jngermon** [comment]
Bad news.... I d'ont know Tutum and deployment process so I can't give you a solution easily. I think that it's a minor PR and we can merge again it in the futur.

**Comment by sameersbn** [comment]
reverted!

**Pull request merged**